relaybot.py

```diff
@@ -52,7 +53,10 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        if options['ssl']:

```

Comment:
How about something like

    mapping = {"True": True, "False": False}
    def convert(value):
        if value in mapping:
            return mapping [value]
        raise TypeError("Cannot convert '{}' to boolean.".format(value))

I say that instead of just `value == "True"` to avoid something like "on"
becoming False.

On Fri, May 1, 2015, 5:21 AM Michael Donaghy <notifications@github.com>
wrote:

> In relaybot.py
> <https://github.com/Thynix/RelayBot/pull/1#discussion_r29494892>:
>
> > @@ -52,7 +53,10 @@ def get(option):
> >              options["nickServPassword"] = get("nickServPassword")
> >
> >          factory = factory(options)
> > -        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
> > +        if options['ssl']:
>
> That'd probably be better. boolean(options['ssl']) would still be
> specified-or-not - what's the pythonic way to turn a string into a boolean
> in "do what I mean" fashion?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Thynix/RelayBot/pull/1/files#r29494892>.
>